app/src/main/java/com/cornellappdev/score/model/Game.kt

```diff
@@ -7,5 +7,60 @@ data class GameCardData(
     val date: String,
     val location: String,
     val genderIcon: Int,
-    val sportIcon: Int,
+    val sportIcon: Int
 )
+
+// Scoring information for a specific team, used in the box score
+data class TeamScore(
+    val team: Team,
+    val scoresByPeriod: List<Int>,
+    val totalScore: Int
+)
+
+// Aggregated game data showing scores for both teams
+data class GameData(
+    val teamScores: Pair<TeamScore, TeamScore>
+)
+
+// Individual scoring event details for the scoring summary
+data class ScoreEvent(
+    val id: Int,             // Unique identifier for each event
+    val time: String,        // Event time (e.g., "6:21")
+    val quarter: String,     // Quarter (e.g., "1st Quarter")
+    val team: Team,          // Team object that includes name and logo URL
+    val eventType: String,   // Event type (e.g., "Field Goal", "Touchdown")
+    val score: String,        // Current score after the event (e.g., "10 - 7")
+    val description: String? = null )
```

Comment:
Plz delete these comments 😅

Reply:
(The others ABOVE each data class are fine; plz convert those to javadoc comments actually to make them useful!)